lib/ngComponents.js

```diff
@@ -0,0 +1 @@
+/*** AUTO-GENERATED BY GULP ***/
```

Comment:
No, I was trying to say that these files will work just as templates for the generation in this repo. When the module is installed in an app, these templates will be instantiated with the real data, but inside the node_modules of the app. Maybe we can just change their names in order to clearify (e.g. ngConstantsTemplate.js will generate ngConstants.js, etc).